### PR TITLE
add libini

### PR DESCRIPTION
**Pull request by rgetz**
This will compete with #405, and only one should be committed to master.

I prefer this one. It's easy, simple and will be maintained.

### CODE REVIEW

File: local.c [hunk 1]
```diff
@@ -19,6 +19,10 @@
 #include "debug.h"
 #include "iio-private.h"
 #include "sort.h"
+#ifdef WITH_LOCAL_CONFIG
+#include "libini/ini.h"
```
Comment (commodo) [inline local.c:23]: does clang-format have a preference for `#include "headers"` to be added after `#include <headers>`?
maybe it's a slightly better idea to move all this block after this;

but we can leave it as is for now;

**Comment by pcercuei** [comment]
If you don't want a submodule, have a look at [git-subrepo](https://github.com/ingydotnet/git-subrepo).

**Comment by dNechita** [comment]
A local copy seems the simplest way to me.

**Comment by rgetz** [comment]
Ok I will merge this by end of today unless anyone has objections.

**Pull request merged**